homeassistant/components/google_maps/device_tracker.py

```diff
@@ -30,7 +32,10 @@
 
 CREDENTIALS_FILE = ".google_maps_location_sharing.cookies"
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+# the parent "device_tracker" have marked the schemas as legacy, so this
+# need to be refactored as part of a bigger rewrite.
+# ignoring the mypy error to allow mypy to check the rest of the integration.
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(  # type: ignore
```

Comment:
This comment and ignore are incorrect. It complains as the platform schema has been marked final.

```suggestion
PLATFORM_SCHEMA = PLATFORM_SCHEMA_BASE.extend(
```

And adjust the import in the top to:

```py
from homeassistant.components.device_tracker import (
    PLATFORM_SCHEMA as PLATFORM_SCHEMA_BASE,
    SOURCE_TYPE_GPS,
)
```

Reply:
Done, I must have been sleeping !